blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java

```diff
@@ -48,6 +49,7 @@
     private ScheduleContext scheduleCtx;
     private long serverTime;
     private static final Timer rollupLocatorExecuteTimer = Metrics.timer(RollupService.class, "Locate and Schedule Rollups for Slot");
+    private static final Histogram locatorsPerShard = Metrics.histogram(RollupService.class, "Locators Per Shard");
```

Comment:
Should the class be `RollupService.class`? I guess we can keep it that way to be consistent.